core/domain/story_jobs_one_off_test.py

```diff
@@ -143,8 +158,10 @@ def test_migration_job_converts_old_story(self):
         topic_services.add_canonical_story(
             self.albert_id, self.TOPIC_ID, self.STORY_ID)
         story = (
-            story_fetchers.get_story_by_id(self.STORY_ID))
+            story_models.StoryModel.get(self.STORY_ID))
         self.assertEqual(story.story_contents_schema_version, 1)
+        story = story_fetchers.get_story_by_id(self.STORY_ID)
```

Comment:
This test shouldn't be done here, This file is only for the one off job test. The migration function need only be tested in story_services_test.

Reply:
I have removed it.